src/fidesops/util/saas_util.py

```diff
@@ -165,3 +167,21 @@ def format_body(
         raise FidesopsException(f"Unsupported Content-Type: {content_type}")
 
     return headers, output
+
+
+def assign_placeholders(value: str, param_values: Dict[str, Any]) -> Optional[str]:
+    """
+    Finds all the placeholders (indicated by <>) in the passed in value
+    and replaces them with the actual param values
+
+    Returns None if any of the placeholders cannot be found in the param_values
+    """
+    if value and isinstance(value, str):
```

Comment:
`value` will always be a string, since it's type is asserted in the function signature. We only have to account for `""`.
```suggestion
    if value:
```

Reply:
I see you added `int` as a possible type for `value`, but this line means the function only handles string `value`s. There's no handling for `int`s that I can see. Please add comprehensive test coverage for this function.